great_expectations_cloud/logging/README.md

```diff
@@ -0,0 +1,32 @@
+# Logging
+
+The GX Agent offers two types of logging by default. It also allows users to pass in a logging configuration file.
+
+## 1. Stdout logging
+
+The default log level for output to stdout is WARN.
+
+Users can provide an optional `--log-level` argument to specify a different logging level for stdout if desired; e.g., `--log-level=info`
+
+These logs can be accessed using the `docker logs {container_name}` command. They are available both when a container is running and after it is exited.
+
+## 2. Debug log files
+
+By default, the GX Agent creates temporary log files inside of running Docker containers. These files can be provided to the Great Expectations Support team if debugging assistance is needed.
+
+Users can provide an optional `--skip-log-file` argument to skip saving log output to files if desired.
+
+A new log file is generated each day with a timestamped filename. The most recent 30 files are kept on a rotating basis.
+
+These files are saved to a `logs` directory in the Docker container and are not accessible after a container has been exited.
+
+Users can access the `logs` directory inside of a running container with the following command(s):
+
+* To copy the logs directory to your local machine: `docker cp {dockerImageId}:/app/logs ~/Desktop/logs`
+* To access the logs directory in the container: `docker exec -it {dockerImageName} /bin/bash`
```

Comment:
@nicgrayson , I know you had mentioned giving users a command to copy all files into one file to send to GX, but since the filenames are timestamped, I think it's better to just use the `docker cp` command. But I left in the `exec -it` command in case it's helpful